thavalon-webapp/src/utils/AccountManager.tsx

```diff
@@ -372,4 +372,84 @@ export class AccountManager {
         }
         return httpResponse;
     }
+
```

Comment:
code below duplicated from above?

Reply:
How this is not a compiler error is beyond me, but nice catch.